Slim/Schema/Contributor.pm

```diff
@@ -85,8 +85,33 @@ sub contributorRoles {
 	return @contributorRoles;
 }
 
+sub isDefaultContributorRole {
+	my $self = shift;
+	my $role = shift;
+	
+	return __PACKAGE__->typeToRole($role) < MIN_CUSTOM_ROLE_ID;
+}
+
 sub defaultContributorRoles {
-	return grep { __PACKAGE__->typeToRole($_) < MIN_CUSTOM_ROLE_ID } contributorRoles();
+	return grep { __PACKAGE__->isDefaultContributorRole($_) } contributorRoles();
+}
+
+sub splitDefaultAndCustomRoles {
+	my $self = shift;
```

Comment:
`$class`

Reply:
When to use $`class` and when to use `$self`?

Reply:
Oh, I forgot to send off the longer comment... just a minute!